ci/goreleaser/goreleaser-fips.yml

```diff
@@ -0,0 +1,294 @@
+# Generated by: gromit policy
+# Generated on: Wed Dec 14 19:54:02 UTC 2022
+
+# Check the documentation at http://goreleaser.com
+# This project needs CGO_ENABLED=1 and the cross-compiler toolchains for
+# - arm64
+# - amd64
+builds:
+  - id: std-linux
+    flags:
+      - -tags=goplugin,boringcrypto
+    ldflags:
+      - -X github.com/TykTechnologies/tyk/gateway.VERSION={{.Version}} -X github.com/TykTechnologies/tyk/gateway.Commit={{.FullCommit}} -X github.com/TykTechnologies/tyk/gateway.buildDate={{.Date}} -X github.com/TykTechnologies/tyk/gateway.builtBy=goreleaser
+    goos:
+      - linux
+    goarch:
+      - amd64
+    binary: tyk
+  - id: std-arm64
```

Comment:
Since boringcrypto is not supported on arm in 1.16, might as well remove all references to arm builds?